### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by glicht**
## Status
Ready

## Description
Documentation updates with focus on contributors.

## Additional changes
PR template update: removed the related branches.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,55 +2,69 @@
...
-
-After you finish developing, there are two steps left before you can create a pull request:
-
- - Follow code conventions [here](https://github.com/demisto/content/tree/master/docs/code_conventions)
- - Run linting and test checks as detailed [here](https://github.com/demisto/content/tree/master/docs/linting)
- - Document your changes in the relevant changelog file as detailed [here](https://github.com/demisto/content/tree/master/docs/release_notes).
- - Validate files are formatted according correctly, by running from the Content root directory: ```PYTHONPATH="`pwd`:${PYTHONPATH}" python2 Tests/scripts/validate_files.py```
- - Make sure you have test playbook [here](https://github.com/demisto/content/tree/master/docs/tests)
- - Make sure you have documentation [here](https://github.com/demisto/content/tree/master/docs/integration_documentation)
+Please read the following guidelines. Following these guidelines will maximize the chances for a fast, easy and effective review process for everyone involved. If something is not clear, please don't hesitate to reach out to us via GitHub, Slack, or email.
```
Comment (Itay4) [inline CONTRIBUTING.md]: I would add the link to difr and the email here instead of in the bottom

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,55 +2,69 @@
...
+
+* Setup a development environment by following the [Getting Started Guide](docs/getting_started).
+* Use the [Package Directory Structure](docs/package_directory_structure) for all Python code. If working on existing code, beyond trivial changes, we require converting to this structure as it allows running linting, unit tests and provides a clearer review process.
+* Make sure to read and follow [code conventions](docs/code_conventions).
+* Run and verify that the various linters we support pass as detailed [here](docs/linting).
+* For Scripts/Integrations make sure to create unit tests as documented [here](docs/tests/unit-testing)
```
Comment (Itay4) [inline CONTRIBUTING.md]:
```suggestion
* For Scripts/Integrations written in Python, make sure to create unit tests as documented [here](docs/tests/unit-testing)
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,55 +2,69 @@
...
+* Create a test playbook as documented [here](docs/tests). **Note**: for small Scripts that have unit tests, a test playbook is optional.
```
Comment (Itay4) [inline CONTRIBUTING.md]: not sure if `small Scripts` is the way to describe these, maybe "simple" or "straight-forward"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,55 +2,69 @@
...
+* Validate that our validation hooks pass. If you used `.hooks/bootstrap` as documented in [Getting Started](docs/getting_started) the validation hook will run automatically upon commit. You can also run the validation hooks manually by running `.hooks/pre-commit`.
+* Document you integration a detailed [here](docs/integration_documentation).
```
Comment (Itay4) [inline CONTRIBUTING.md]:
```suggestion
* Document your integration as detailed [here](docs/integration_documentation).
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -2,55 +2,69 @@
...
+* Document your changes in the relevant changelog file as detailed [here](docs/release_notes).
+  
  - Circle CI build must be green [here](https://github.com/demisto/content/tree/master/docs/tests/circleci)
```
Comment (Itay4) [inline CONTRIBUTING.md]:
```suggestion
 - Circle CI build must be green. See further details [here](https://github.com/demisto/content/tree/master/docs/tests/circleci)
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -2,55 +2,69 @@
...
 
+## Review Process
+Once you open a pull request we have a bot (`@content-bot`) that will assist you and provide comments and reminders about the state of the pull request. Additionally, a member of the team will be assigned to review the pull request. Comments will be provided by the team member as the review process progresses.
+
+You will see a few [GitHub Status Checks](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks) that help validate that your pull request is according to our standards:
+
+* **ci/circleci: build**: We use [CircleCI](https://circleci.com/gh/demisto/content) to run a full build on each commit of your pull request. The build will run our content validation hooks, linting and unit test. We require that the build pass (green build). If it is failing our content-bot will also add instructive comments about what sections are failing in the build. Follow the `details` link of the status to see the full build UI of CircleCI. More details about our build flow of CircleCI is available [here](docs/tests/circleci).
```
Comment (Itay4) [inline CONTRIBUTING.md]:
```suggestion
* **CI build**: We use [CircleCI](https://circleci.com/gh/demisto/content) to run a full build on each commit of your pull request. The build will run our content validation hooks, linting and unit test. We require that the build pass (green build). If it is failing our content-bot will also add instructive comments about what sections are failing in the build. Follow the `details` link of the status to see the full build UI of CircleCI. More details about our build flow of CircleCI is available [here](docs/tests/circleci).
```

File: docs/code_conventions/README.MD [hunk 7]
```diff
@@ -1,6 +1,12 @@
-# Overview
-We use standardized code conventions to ensure uniformity across all Demisto integrations. This section outlines some of these code conventions.
+# Python Code Conventions
+We use standardized code conventions to ensure uniformity across all Demisto Integrations. This section outlines our code conventions.
 
+New integrations and scripts should follow these conventions. When working on small fixes and modifications to existing code, follow the conventions used in the existing code.
+
+**Note:** Demisto supports also Javascript integrations and scripts. Our preferred development language is Python and all new integrations and large scripts should be developed in Python, which also provides a wider set of capabilities compared to the available Javascript support. Small scripts may still be developed in Javascript using the conventions provided by the default script template used in the Demisto IDE.
```
Comment (Itay4) [inline docs/code_conventions/README.MD]:
```suggestion
**Note:** Demisto supports also JavaScript integrations and scripts. Our preferred development language is Python, and all new integrations and large scripts should be developed in Python, which also provides a wider set of capabilities compared to the available JavaScript support. Small scripts may still be developed in JavaScript using the conventions provided by the default script template used in the Demisto IDE.
```

Comment (Itay4) [inline docs/code_conventions/README.MD]: same comment about the "small scripts"

File: docs/code_conventions/README.MD [hunk 8]
```diff
@@ -1,6 +1,12 @@
...
+
+## Python 2 vs 3
```
Comment (Itay4) [inline docs/code_conventions/README.MD]:
```suggestion
## Python 2 vs Python 3
```

File: CONTRIBUTING.md (hunk 6 above)
Comment (glicht) [reply CONTRIBUTING.md]: I want to use the same names we show at status checks in Github

File: docs/code_conventions/README.MD (hunk 8 above)
Comment (glicht) [reply docs/code_conventions/README.MD]: I think this is ok

File: CONTRIBUTING.md (hunk 6 above)
Comment (Itay4) [reply CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md (hunk 5 above)
Comment (glicht) [reply CONTRIBUTING.md]: This was old. I moved this to the review process status checks section.

**Comment by Itay4** [comment]
@glicht lets also add link to the HelloWorld integration/script

File: README.md [hunk 9]
```diff
@@ -30,69 +34,62 @@ Integrations written in Javascript or Python enable the Demisto Platform to orch
...
 
 ### Context Standards
 When we are working with data that is generic across all platforms, we format them according to our context standards. This helps integrations work interchangeably inside other playbooks.
 
-[Learn about our Context Standards here](https://github.com/demisto/content/blob/master/docs/context_standards)
+[Learn about our Context Standards here](docs/context_standards)
 
 ### Docker
-In some cases, it will be necessary to create a docker image to enable your integration to run. When this happens, we must create a new docker image using the steps outlined here:
+We use docker to run python scripts and integrations in a controlled environment. You can configure an existing docker image form the [Demisto Docker Hub Organization](https://hub.docker.com/u/demisto/) or create a new docker image to suite your needs. More about how to use Docker is available here:
```
Comment (yaakovi) [inline README.md]:
```suggestion
We use docker to run python scripts and integrations in a controlled environment. You can configure an existing docker image from the [Demisto Docker Hub Organization](https://hub.docker.com/u/demisto/) or create a new docker image to suite your needs. More information about how to use Docker is available here:
```

**Comment by glicht** [comment]
Added to code conventions reference to HelloWorld and tempates.

**Comment by anara123** [comment]
looks great

**Pull request merged**